examples/emotiv_oscope_local.py

```diff
@@ -0,0 +1,64 @@
+"""
+Simple demonstration of streaming EEG data from an Emotiv device to a QOscilloscope
+viewer, for linux case.
+
+Both device and viewer nodes are created locally without a manager.
+"""
+
+from pyacq.devices.eeg_emotiv import Emotiv
+from pyacq.viewers import QOscilloscope
+import pyqtgraph as pg
+import os
+
+
```

Comment:
I suggest something like:

```
import sys
usage = """Usage:
    python %s <device>
where <device> is the name of the Emotiv device file (like /dev/hidraw3) on linux,
or _____ on windows.\n
""" % sys.argv[0]

if len(sys.argv) != 2:
    sys.stderr.write(usage)
device_handle = sys.argv[1]
```

This should allow the example to support windows while also allowing for different device names on linux (I assume it is unlikely that other machines will assign the device to `/dev/hidraw3` as well).